deploy/runtime/ui/Battery Voltage.txt

```diff
@@ -1359,16 +1447,25 @@ function cell_voltage_vs_dod()
 
 		//logmsg('Vfull: ' + Vfull + ' Vexp: ' + Vexp + ' Vnom: ' + Vnom + ' Qfull: ' + Qfull + ' Qexp: ' + Qexp + ' Qnom: ' + Qnom + 'Qnom_percent: ' + Qnom_percent + ' C: ' + C_rate); 
 		//logmsg('A: ' + A + ' B: ' + B + ' K: ' + K + 'R: ' + R + ' EO: ' + E0 ); 
-
+		//I = Qfull_cutoff * C_rate;
+		//I_v = Qfull_cutoff * C_rate;
 		for (i = 1; i <= stop; i++){
 			count++;
-			t = i*0.1;
+			t = i*0.10;
 			q0 = Qfull - I*t;
-			V = E0 - R*I - K*(Qfull/(Qfull-I*t)) + A*exp(-B*I*t);
+			V = E0 - R*I - K*(Qfull_cutoff/(Qfull_cutoff -I*t))+ A*exp(-B*I*t);
```

Comment:
The formula in SSC looks like it has one more instance of Qfull than this one. Is that true or am I missing something?